charts/stable/jellyfin/questions.yaml

```diff
@@ -82,7 +82,12 @@ questions:
       type: dict
       attrs:
 # Include{fixedEnv}
-
+        - variable: JELLYFIN_PublishedServerUrl
+          description: "Published Server URL"
+          label: "The Server URL to publish in UDP Auto Discovery response."
+          schema:
+            type: string
+            default: ""
```

Comment:
We should keep a mental note about setting this automatically based on LoadbalancerIP, nodeIP or Ingess.